apps/supernova/turbo.json

```diff
@@ -3,23 +3,23 @@
   "tasks": {
     "dev": {
       "dependsOn": [
-        "@cloudoperators/juno-ui-components#build",
```

Comment:
Why did you remove the turbo dependency to `ui-components`?

Reply:
good catch that was unintentional.